felix/cmd/calico-bpf/commands/policy_debug.go

```diff
@@ -55,7 +55,7 @@ var policyDumpCmd = &cobra.Command{
 		}
 		hooks := []string{}
 		if hook == "all" {
-			hooks = []string{"ingress", "egress"}
+			hooks = []string{"ingress", "egress", "xdp"}
```

Comment:
This can be replaced by `bpf.Hooks`: https://github.com/projectcalico/calico/blob/57b84fa83985a288ea7c2bddb858e5014ab5705f/felix/bpf/bpf.go#L60